framework.go

```diff
@@ -359,6 +359,26 @@ func (f *Framework) GetAppAndValues(ctx context.Context, name, namespace string)
 	return app, values, nil
 }
 
+// GetKubeadmControlPlane returns the KubeadmControlPlane resource
```

Comment:
Can we add a note in the docs here about if managed control plane this will return nil?

Reply:
Added a comment for the managed clusters.